Sources/CvPlot.cpp

```diff
@@ -16105,21 +15805,14 @@ bool CvPlot::isInUnitZoneOfControl(PlayerTypes ePlayer) const
 {
 	for (int iI = 0; iI < NUM_DIRECTION_TYPES; ++iI)
```

Comment:
FYI There is an range for this, which automatically skips invalid plots: CvPlot::adjacent_range
`foreach_(CvPlot* adjacentPlot, adjacent())`